python/nav/web/info/forms.py

```diff
@@ -15,41 +15,49 @@
 # along with NAV. If not, see <http://www.gnu.org/licenses/>.
 #
 """Forms for use in the info subsystem"""
-
+from crispy_forms.templatetags.crispy_forms_field import css_class
 from django import forms
-from crispy_forms.helper import FormHelper
-from crispy_forms_foundation.layout import Layout, Row, Column, Submit, Field
+
+from nav.web.crispyforms import (
+    set_flat_form_attributes,
+    SubmitField,
+    FormRow,
+    FormColumn,
+)
 
 
 class SearchForm(forms.Form):
     """The searchform used for base searches"""
 
     query = forms.CharField(max_length=100, label='', required=False)
 
-    def __init__(self, *args, **kwargs):
-        self.helper = get_formhelper(
-            kwargs.pop('form_action', ''), kwargs.pop('placeholder', 'Search')
+    def __init__(
+        self, *args, form_action: str = '', placeholder: str = 'Search', **kwargs
+    ):
+        super().__init__(*args, **kwargs)
+        self.attrs = set_flat_form_attributes(
+            form_action=form_action,
+            form_method='get',
+            form_class='search-form',
+            form_fields=[
+                FormRow(
+                    [
+                        FormColumn([self['query']], css_classes='medium-9'),
+                        FormColumn(
+                            [
+                                SubmitField(
+                                    value='Search', css_classes='submit button postfix'
+                                )
+                            ],
```

Comment:
```suggestion
                            [SubmitField(value='Search', css_classes='postfix')],
```

The other css classes are already added by `SubmitField` 

https://github.com/Uninett/nav/blob/2dae9d3161e49523d0313b0bc57da9e00629446c/python/nav/web/templates/custom_crispy_templates/submit_field.html#L7